src/tab.js

```diff
@@ -130,7 +130,11 @@ SideTab.prototype = {
     this._setIcon("img/defaultFavicon.svg");
   },
   setSpinner() {
-    this._setIcon("img/loading-spinner.svg");
+    // Workaround until context-stroke is enabled in sidebar by default.
+    const isDarkTheme = document.body.classList.contains("dark-theme");
+    const isShrinked = document.getElementById("tablist").classList.contains("shrinked");
+    const frag = isDarkTheme && isShrinked && "#loading-invert" || "";
```

Comment:
> But if you want the favicon as a foreground and put the spinner as a background, both icons will be shown.

Do we actually want that?

> Also, even if all the favicons are placed as background images, in that case, it will be necessary to generate CSS dynamically

The site favicon should be set using JS yes (like we already do), instead of using src we would use el.style.backgroundImage.